### DISCUSSION TITLE
How to launch a Gazebo Garden simulation with PX4?

### DISCUSSION THREAD

**Discussion by Cristian-wp** [aerostack2/aerostack2#715] [Q&A]
Hello, I am trying to figure out how to to launch a PX4 simulation with the new versions of gazebo but I can not understand it.
I have also try to use [project_mavlink](https://github.com/aerostack2/project_mavlink/tree/main) but since PX4 runs in a docker container I can not see the drones fly even if the mission end well.
Plus I wish to use the DDS to get drones topic like it was in [project_px4_vision](https://github.com/aerostack2/project_px4_vision) example.

Can you please give me some help to do this?

**Comment by miferco97** [comment]
check that you are working with the ```web_gui_4uavs``` branch of the project.
Also run the command ```$ xhost +``` before launching the ```launch_sitl.bash``` in order to provide access to the DISPLAY to the gazebo container.
